docs/ddd/dictionary/items.md

```diff
@@ -0,0 +1,680 @@
+| 12  | EXP Walker        |
+| 13  | EXP Zero          |
+| 14  | Damage Syphon     |
+| 15  | Second Chance     |
+| 16  | Once More         |
+| 17  | Scan              |
+| 18  | Leaf Bracer       |
+| 19  | Treasure Magnet   |
+| 20  | Link Critical     |
+| 21  | Support Boost     |
+| 22  | Waking Dream      |
+| 23  | ???               |
+| 24  | ???               |
+| 25  | ???               |
+| 26  | ???               |
+| 27  | ???               |
+| 28  | ???               |
+| 29  | ???               |
+| 30  | ???               |
+| 31  | ???               |
+| 32  | ???               |
+| 33  | ???               |
+| 34  | ???               |
+| 35  | ???               |
+| 36  | ???               |
+| 37  | ???               |
+| 38  | ???               |
+| 39  | ???               |
+| 40  | ???               |
+| 41  | ???               |
+| 42  | ???               |
+| 43  | ???               |
+| 44  | ???               |
+| 45  | ???               |
+| 46  | ???               |
+| 47  | ???               |
+| 48  | ???               |
+| 49  | ???               |
+| 50  | Fleeting Figment  |
+| 51  | Fleeting Fancy    |
+| 52  | Fleeting Fantasy  |
+| 53  | Lofty Figment     |
+| 54  | Lofty Fancy       |
+| 55  | Lofty Fantasy     |
+| 56  | Rampant Figment   |
+| 57  | Rampant Fancy     |
+| 58  | Rampant Fantasy   |
+| 59  | Dulcet Figment    |
+| 60  | Dulcet Fancy      |
+| 61  | Dulcet Fantasy    |
+| 62  | Malleable Fantasy |
+| 63  | Prickly Fantasy   |
+| 64  | Wild Fantasy      |
+| 65  | Epic Fantasy      |
+| 66  | Charming Fantasy  |
+| 67  | Brilliant Fantasy |
+| 68  | Intrepid Figment  |
+| 69  | Intrepid Fancy    |
+| 70  | Intrepid Fantasy  |
+| 71  | Savage Fantasy    |
+| 72  | Noble Figment     |
+| 73  | Noble Fancy       |
+| 74  | Noble Fantasy     |
+| 75  | Grim Figment      |
+| 76  | Grim Fancy        |
+| 77  | Grim Fantasy      |
+| 78  | Vibrant Figment   |
+| 79  | Vibrant Fancy     |
+| 80  | Vibrant Fantasy   |
+| 81  | Troubling Figment |
+| 82  | Troubling Fancy   |
+| 83  | Troubling Fantasy |
+| 84  | Wondrous Figment  |
+| 85  | Wondrous Fancy    |
+| 86  | Wondrous Fantasy  |
+| 87  | HP orb x1         |
+| 88  | HP orb x2         |
+| 89  | HP orb x3         |
+| 90  | Munny orb x1      |
+| 91  | Munny orb x2      |
+| 92  | Munny orb x3      |
+| 93  | Munny orb L x1    |
+| 94  | Drop orb x1       |
+| 95  | Drop orb x2       |
+| 96  | Drop orb x3       |
+| 97  | Drop obr L x1     |
+
+---
+
+## Keyblades
+
+| Id  | Description        |
+| --- | ------------------ |
+| 0   | Kingdom Key        |
+| 1   | Skull Noise (S)    |
+| 2   | Guardian Bell (S)  |
+| 3   | Ferris Gear        |
+| 4   | Dual Disc (S)      |
+| 5   | All for One (S)    |
+| 6   | Counterpoint (S)   |
+| 7   | Sweet Dreams (S)   |
+| 8   | Ultima Weapon (S)  |
+| 9   | Unbound (S)        |
+| 10  | Divewing (S)       |
+| 11  | End of Pain (S)    |
+| 12  | Knockout Punch (S) |
+| 13  | ? (S)              |
```

Comment:
https://github.com/Xeeynamo/KingdomSaveEditor/blob/master/KHSave.LibDDD/Types/EquipmentType.cs#L456 This is what I got from cheat lists for the 3ds version of the game

Reply:
That's where I got my initial list from, but I checked the values and I'm using the names the game itself displays (PC). I don't know where "HOLLOW_DRAGONITE" got those names from.